chsdi/templates/htmlpopup/swissboundaries_bezirk.mako

```diff
@@ -1,8 +1,13 @@
 <%inherit file="base.mako"/>
 
 <%def name="table_body(c,lang)">
+<%
+linkeddatahost = request.registry.settings['linkeddatahost']
+ldlink = linkeddatahost + '/boundaries/district/' + str(c['featureId'])
```

Comment:
if I may suggest, featureId is mainly used for backward compatibility issues, I would like to get rid of these duplicated ids at some point, so always prefer id.